jetty-io/src/main/java/org/eclipse/jetty/io/EndPoint.java

```diff
@@ -215,7 +232,26 @@ default void close()
      * destination (ie is not buffering any data).
      * @throws IOException If the endpoint is closed or output is shutdown.
      */
-    boolean flush(ByteBuffer... buffer) throws IOException;
+    default boolean flush(ByteBuffer... buffer) throws IOException
+    {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * <p>Sends to the given address the data in the given buffers.</p>
+     * <p>This methods should be used to send UDP data, in conjunction
+     * with {@link DatagramChannelEndPoint}.</p>
+     *
+     * @param address the peer address to send data to
+     * @param buffers the buffers containing the data to send
+     * @return true if all the buffers have been consumed
+     * @throws IOException if the send fails
+     * @see #write(Callback, SocketAddress, ByteBuffer...)
+     */
+    default boolean send(SocketAddress address, ByteBuffer... buffers) throws IOException
```

Comment:
I don't really understand this signature.    If `send` returns false and the buffers have only been partially consumed, what has happened and what am I supposed to do?  Have a datagram with the partial content been send?   Should I then call it again but with a null address?   Will the subsequent calls send new datagrams or just complete the first? 

At the very least this needs to be called `flush` and have better javadoc, but can't we just hide it and only use the new `write`?